src/Microsoft.AspNet.Hosting.Abstractions/EnvironmentName.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class EnvironmentName
```

Comment:
@Tragetaschen I already implemented as ```enum``` before but when I revise the entire repo I notice that environment name may comes from configuration as string, so that's why I used ```class enum```